docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -38,18 +38,18 @@ The screen reader will announce the value title (`Number of passengers`) and but
 ```jsx
 <Stack>
   <Stack>
-    <Text id="passengers">Passengers</Text>
+    <Text id="adults">Adults</Text>
   </Stack>
   <Stepper
     step={1}
     minValue={0}
     minValue={10}
     ariaLabelValue="Number of passengers"
-    ariaLabelledby="passengers"
+    ariaDescribedby="adults"
     titleDecrement="Remove a passenger"
     titleIncrement="Add a passenger"
   />
 </Stack>
 ```
 
-This example includes `ariaLabelledby` prop. In this case, `ariaLabelledBy` prop is prioritized over `ariaLabelValue`, so the screen reader will announce the value title (`Passengers`) and buttons title (`Add a passenger`, `Remove a passenger`) once they are focused by the screen reader.
+This example includes `ariaDescribedby` prop. Value of `ariaDescribedby` prop is used to announce the complementary information about the Stepper component title (`Adults`) once the Stepper button is focused by the screen reader.
```

Comment:
Here, as you added `aria-describedby` for both buttons and input, I guess it makes sense to mention it - because it's not announcing complementary information only when Stepper button is focused, but also other two elements.